rest/kadai-rest-spring/src/main/java/io/kadai/common/rest/ldap/LdapClient.java

```diff
@@ -840,6 +848,179 @@ void testMinSearchForLength(final String name) throws InvalidArgumentException {
     }
   }
 
+  private AccessIdRepresentationModel lookupAndDebugLog(
```

Comment:
The fact that we write debug logs should not be in the operation name, shouldn't it? It is rather a side effect depending on the log level.

Reply:
I prefer to be as precise as possible in a method name. In this was the reader doesn't have to read the method definition. But you also have a point, so I removed "AndDebugLog" from the "search" and "lookup" methods.